packages/mantine/src/components/browser-preview/BrowserPreview.tsx

```diff
@@ -0,0 +1,71 @@
+import {InfoSize16Px} from '@coveord/plasma-react-icons';
+import {
+    Box,
+    ColorSwatch,
+    DefaultProps,
+    Flex,
+    Group,
+    Selectors,
+    Stack,
+    Text,
+    Title,
+    Tooltip,
+    useMantineTheme,
+} from '@mantine/core';
+import {PropsWithChildren} from 'react';
+import useStyles from './BrowserPreview.styles';
+
+export interface BrowserPreviewProps extends DefaultProps<Selectors<typeof useStyles>> {
+    /**
+     * Text to display in a tooltip in the header.
+     *
+     */
+    headerTooltip?: string;
+    /**
+     * Custom title to be displayed at the center of the header.
+     */
+    title?: string;
+}
+
+export const BrowserPreview = ({
+    children,
+    headerTooltip,
+    title,
+    // Style props
+    classNames,
+    className,
+    styles,
+    unstyled,
+}: PropsWithChildren<BrowserPreviewProps>) => {
```

Comment:
Should we spread the rest on the root element?

```suggestion
export const BrowserPreview = ({
    children,
    headerTooltip,
    title,
    // Style props
    classNames,
    className,
    styles,
    unstyled,
    ...others,
}: PropsWithChildren<BrowserPreviewProps>) => {
```